pages/common/conda.md

```diff
@@ -0,0 +1,35 @@
+# conda
+
+> Package, dependency and environment management for any programming language.
+
+- Create a new environment, installing named packages (eg. python=2.7):
```

Comment:
Ah ok.. This is actually how I first wrote it, but then I changed it before submitting the pull request. I wasn't sure if it would be confusing as it would be difficult to distinguish between named parameters/flags and example strings? Are there any other _tldr_ pages that you know of which do this? How would you propose that we write this?